extensions/arc/package.json

```
@@ -10,7 +10,18 @@
   "categories": [
     "Web",
     "Applications",
-    "Productivity"
+    "Productivity",
+    "Web"
```

```suggestion
    "Productivity"
```
Duplicated "Web" category